src/main/java/se/liu/ida/hefquin/engine/data/impl/VocabularyMappingImpl.java

```diff
@@ -335,7 +352,8 @@ protected SPARQLGraphPattern translatePredicate(final TriplePattern tp){
 		final Set<Triple> mappings = new HashSet<>();
 		final Iterator<Triple> i = vocabularyMapping.find(s,p,o);
 		while(i.hasNext()) {
-			mappings.add(i.next());
+			Triple next = i.next();
```

Comment:
```suggestion
			final Triple next = i.next();
```

Reply:
You can accept this change requests by clicking the button "Commit suggestion", then you can get the changes into your local branch with `git pull`. I recommend this, because it will mark the conversation as resolved and thus make the PR shorter to look at.